lms/djangoapps/certificates/models.py

```diff
@@ -854,16 +860,21 @@ class CertificateGenerationCourseSetting(TimeStampedModel):
     for the course.  This is enforced in the UI layer, but
     not in the data layer.
 
+    'language_specific_templates' controls whether certificates for the
+    course should be generated using specific language templates.
+
     """
     course_key = CourseKeyField(max_length=255, db_index=True)
-    enabled = models.BooleanField(default=False)
+    enabled = models.BooleanField(default=False)  # Deprecated
+    can_self_generate = models.BooleanField(default=False)
+    language_specific_templates = models.BooleanField(default=False)
```

Comment:
Nit: a clearer name might be `can_use_language_specific_templates`. It's also more consistent with the naming of the other field.